app/util/data_preparation/crowd_prepare_data.py

```diff
@@ -0,0 +1,96 @@
+import urllib3
+import random
+import string
+
+from util.conf import CROWD_SETTINGS
+from util.api.crowd_clients import CrowdRestClient
+from util.project_paths import CROWD_USERS
+
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+USERS = "users"
+DEFAULT_USER_PASSWORD = 'password'
+DEFAULT_USER_PREFIX = 'performance'
+USER_SEARCH_CQL = f'name={DEFAULT_USER_PREFIX}*'
+ERROR_LIMIT = 10
+
+NECESSARY_USERS_COUNT = 10000
```

Comment:
What is that? Looks like this variable has to be renamed.

Reply:
Fixed